Sources/Controllers/2.0/OAFavoriteViewController.mm

```diff
@@ -118,6 +111,19 @@ - (id) initWithLocation:(CLLocationCoordinate2D)location andTitle:(NSString*)for
         self.topToolbarType = ETopToolbarTypeMiddleFixed;
     }
     return self;
+
+}
+
+- (void) setupCollapableViews:(OAFavoriteItem *)favorite
```

Comment:
Why do we need it in OAFavoriteViewController.mm? All things related to collapsable coordinates view should be in OAEditTargetViewController.mm